### ISSUE TITLE
No commenting when interaction is only "recent-posts"

### ISSUE THREAD

**Issue by MelBourbon** [alexal1/Insomniac#341]
I want to enable the commenting only in sessions where the bot is interacting only with "recent-posts" to be sure that the comment is fitting to the post. But I noticed that the bot is not commenting at all in "recent-posts" interactions. Is this intended? Can this be changed? It seems only to work in "recent-likers" interactions.

**Comment by simonmannsfeld** [comment]
I agree to @MelBourbon. It feels like an issue. I am fully supporting the way of use from @MelBourbon and want to use it this way, too.

Unfortunately I have no time to fix this bug until Friday.

Maybe someone can help a bit earlier. 😊

**Comment by Fadude** [comment]
Currently, only likes are supported under '*-posts' target.
Following & Commenting will come in future
